build_node/build_node_builder.py

```diff
@@ -57,11 +62,13 @@ def __init__(self, config, thread_num, terminated_event,
         graceful_terminated_event : threading.Event
             Shows, if process got "kill -10" signal.
         """
-        super(BuildNodeBuilder, self).__init__(name='Builder-{0}'.format(
-            thread_num))
+        super(BuildNodeBuilder, self).__init__(
```

Comment:
In modern python we can do `super().method(arg)`, see https://docs.python.org/3/library/functions.html#super

Reply:
same as the above for string formatting, currently it's everywhere in older code. we could do one task for all reformatting to update the codebase